### PR TITLE
feat(*): Add all of the objects for the schema

### PR DESCRIPTION
**Pull request by chriswmackey**
This is a commit for all of the objects in the schema (the only one currently in the repo is Vector3D).

I could particularly use feedback on the display_modes and line_types.

### CODE REVIEW

**Comment by chriswmackey** [comment]
Hey @MingboPeng ,

Yes, you can find some sample OpenAPI schemas here:
https://drive.google.com/drive/folders/1GNSHcDF3ghfondGi5ke7w0Oz6pRS-F4-?usp=sharing

The README also has a sample of how to generate the docs/openapi specification from the cloned repo.

File: ladybug_display_schema/display3d.py [hunk 1]
```diff
@@ -14,7 +18,196 @@ class DisplayVector3D(DisplayBaseModel):
...
+        ...,
+        description='Mesh3D for the geometry.'
+    )
+
+    colors: List[Color] = Field(
+        ...,
+        description='A list of colors that correspond to either the faces '
+        'of the mesh or the vertices of the mesh. It can also be a single color '
+        'for the entire mesh.'
+    )
```
Comment (mostaphaRoudsari) [inline ladybug_display_schema/display3d.py:132]: I think we should add a Pydantic validator for this.

1. check the lengths of the inputs and ensure it is either equal to the number of faces or vertices.
2. if there is only a single color, generate a list with the length of the faces.

**Review by mostaphaRoudsari — APPROVED** [review]
Hey @chriswmackey! I have no major comments. I t think the object with colors can benefit from an additional validator.

**Review by MingboPeng — APPROVED** [review]

File: ladybug_display_schema/display3d.py (hunk 1 above)
Comment (chriswmackey) [reply ladybug_display_schema/display3d.py:132]: Done!

**Pull request merged**